spec/support/corpus.rb

```diff
@@ -29,9 +29,11 @@ class Project
       START_MESSAGE               = 'Starting - %s'.freeze
       FINISH_MESSAGE              = 'Mutations - %4i - %s'.freeze
 
+      DEFAULT_MUTATION_COUNT = 0
```

Comment:
I came to the conclusion we should freeze integer literals also, to prevent accidental modification of their singleton class.